src/renderer/cluster-modals/cluster-modals-registrator.injectable.ts

```diff
@@ -0,0 +1,25 @@
+import { getInjectable } from "@ogre-tools/injectable";
+import { extensionRegistratorInjectionToken } from "../../extensions/extension-loader/extension-registrator-injection-token";
+import type { LensRendererExtension } from "../../extensions/lens-renderer-extension";
+
+const clusterModalsRegistratorInjectable = getInjectable({
+  id: "cluster-modals-registrator",
+
+  instantiate: (): ExtensionRegistrator => {
+    return (ext) => {
+      const extension = ext as LensRendererExtension;
+
+      return extension.clusterModals.map((registration) => {
+        return {
+          id: registration.id,
+          Component: registration.component,
+          visible: registration.visible,
+        }
```

Comment:
Have you tried that this really works? Seems to me this map should return an array of injectables, but this is an array of unrelated objects.

Additionally: I really wonder why typing in `extensionRegistratorInjectionToken` doesn't guard against this. Check?

Reply:
Fixed behaviour. Tested both in real-world and in tests as well.